### ISSUE TITLE
render groups individually in group view

### ISSUE THREAD

**Issue by evs-pn** [evait-security/envizon#2]
All groups are always fully rendered in the group view (including datatables) initially and on each change (move, copy, etc.). Performance suffers dramatically with a high amount of groups (> ~100).
Each group should be rendered individually once selected in the sidebar and should be removed (not only made invisible) from the html when clicking on the "close"-button. All visible groups should be re-rendered if any change (move, copy, etc.) happens. In order to implement this enhancement, the partial render group stuff has to be changed/rewritten. Furthermore a js-component needs to be written in order to select and re-render each visible group.

tl;dr

The performance with a high number of groups will improve in the future. We are on it!

**Comment by evs-pn** [comment]
Done. We just decide that only one group can be selected at the same time in order to keep the interface clean. On every click in the sidebar or change (move, delete, etc.) the selected group will be fully rerendered. If you are view a group and start a global search, the selected group will be replaced by the search content if you click on "Add TO GROUPP-VIEW".
